### ISSUE TITLE
A11y: Hint texts for the current password not linked to the input fields

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7031]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The information text “Please enter your current password to confirm the changes.” (CSS selector .text-muted) is not linked to the corresponding password input.

Screen readers require a semantic link between the input field and the text in order to read the text for the input field to blind users when navigating to it.

*_Expected behaviour:_*

These texts must be linked to the corresponding password input field via the `aria-describedby` attribute.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. navigate to /account/profile
2. navigate to "Change password"

![Image](https://github.com/user-attachments/assets/c8f8af2a-9884-437a-804d-156992d2919b)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40808**



[created from NEXT-40808, comment 541309]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40808, comment 541563]

**Comment by Phil23** [comment]
See my other comment: https://github.com/shopware/shopware/issues/7030#issuecomment-2696632519
